web/package.json

```diff
@@ -49,6 +49,7 @@
     "postcss-preset-env": "^6.7.0",
     "preload-webpack-plugin": "^3.0.0-beta.3",
     "promise-decode-audio-data": "^0.4.1",
+    "query-string": "^6.8.3",
```

Comment:
I'm weary of adding a new dependency for this, since we're going to strip it out of the code in a few days.